documentation/architecture/decisions/0002-configuration-management.md

```diff
@@ -4,7 +4,11 @@ Date: 2018-04-18
 
 ## Status
 
-Pending
+No longer relevant.
+
+This decision related to the alpha code (in
+alphagov/prometheus-aws-configuration), which is no longer being
```

Comment:
I think the plan was to append `-alpha` eventually to the alpha repo and drop our `-beta` from the beta repo meaning this link may later be incorrect. Definitely not the end of the world and I'm happy for this to go in but may be worth us doing the repo name change soon if we still intend to.

Reply:
i think it may be worth rethinking that plan - I think we should leave the alpha where it (but archive it) and rename the beta to something shorter and maybe more in line with other RE terraform repos? /shrug

Either way, I think renaming the beta to have the same name that the alpha used to have is problematic for the reasons you describe.